src/thunderbots/software/test/ai/world/game_state.cpp

```diff
@@ -344,98 +344,3 @@ TEST_F(GameStateRestartTest, penalty_them_restart_test)
     EXPECT_TRUE(game_state.isPlaying());
     EXPECT_FALSE(game_state.isPenalty());
 }
-
-TEST_F(GameStateRestartTest, direct_free_us_restart_test)
```

Comment:
Just to check, these tests are now captured in the other PREDICATE_TESTs right? (not a blocking comment, just asking for clarification)